cli/turbo.json

```diff
@@ -3,7 +3,34 @@
   "extends": ["//"],
   "pipeline": {
     "build": {
-      "outputs": ["turbo", "turbo.exe"]
+      "outputs": [
+        // TODO: this gets moved over to ../target, can we omit this?
+        "turbo",
```

Comment:
yeah I'm not sure why this is here. We build `go-turbo` in `cli` but not `turbo` anymore

Reply:
probably because it was there before rust